src/lang/inspect/defs.rs

```diff
@@ -308,14 +301,15 @@ pub struct ModuleDef {
 }
 
 impl ModuleDef {
-    /// Constructs new [`ModuleDef`] instance.
+    /// Constructs a new [` ModuleDef `] instance.
```

Comment:
Why do some identifiers in docstrings have spaces around them and some don't?

Reply:
Grazie did this to me and I blindly accepted its suggestions 🤦🏻